packages/graphql/src/nexus.ts

```diff
@@ -35,7 +46,7 @@ export const ResponseInput = inputObjectType({
 export const QuestionData = interfaceType({
```

Comment:
using an interface type here allows us to share all of the common properties of a question, while having different types for the options property.